pkg/cli/rack.go

```diff
@@ -104,6 +104,16 @@ func validateParams(params map[string]string) error {
 		return errors.New("to schedule your rack to turn on/off you need both ScheduleRackScaleDown and ScheduleRackScaleUp parameters")
 	}
 
+	// format: "key1=val1,key2=val2"
+	if tags, has := params["tags"]; has {
```

Comment:
hey, nahid, do you think a validation like this it could be done by regex? I know, a lot of people are not fan of regex but it seems cleaner for me.

Reply:
But it's hard to understand what that regex is for by looking
do you have the regex expression for it?

Reply:
Yeah, if it's some obscure regexp I'd rather just keep it like this.
Even with regexp, I think we'd still need to do some type of check to compare the found groups with the actual string.